__tests__/app.test.js

```diff
@@ -0,0 +1,74 @@
+const app = require("../db/app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("app", () => {
+  describe("/api/categories", () => {
+    test("status: 200", () => {
+      return request(app).get("/api/categories").expect(200);
+    });
+    test("responds with an array", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+        });
+    });
+    test("arry of category objects contain the correct properties", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((category) => {
+            expect(category).toHaveProperty("slug", expect.any(String));
+            expect(category).toHaveProperty("description", expect.any(String));
+          });
+        });
+    });
+  });
+
+  describe("/api/reviews", () => {
+    test("status: 200", () => {
+      return request(app).get("/api/reviews").expect(200);
+    });
+    test("responds with a reviews array", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+        });
+    });
+    test("array contains objects with correct properties", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((review) => {
+            expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(Date));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(Number));
+          });
+          // test("comment count property gives correct total count", () => {});
+          // test("reviews sorted in descending order of date", () => {});
+        });
+    });
+  });
+});
```

Comment:
I can see that you're still working on this endpoint - I'd have a branch for each ticket and then you can merge each one separately into main when it is approved. 

Make sure that your endpoints are consistent - the one above is going to '/api/categories' to look for reviews